internal/util/rt/errors/errors.go

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
use full name instead of `rt` for the folder name?

Reply:
Renaming to `reftracker` because `referencetracker` seems a bit too long I think. Please let me know if you had something else in mind.